components/List/examples/MDCSelfSizingStereoCellExample.m

```diff
@@ -40,7 +40,7 @@ - (void)viewDidLoad {
   [super viewDidLoad];
   self.parentViewController.automaticallyAdjustsScrollViewInsets = NO;
   self.automaticallyAdjustsScrollViewInsets = NO;
-  self.listScheme = [[MDCListScheme alloc] init];
```

Comment:
Nit: Please only assign this property if it's `nil`. Otherwise, please leave the injected value as-is because the catalog should be assigning it.